.github/workflows/x-silo.yml

```
@@ -0,0 +1,94 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: xSilo Quality Test
+
+env:
+  FOUNDRY_PROFILE: x_silo
+  RPC_MAINNET: ${{ secrets.RPC_MAINNET }}
+  RPC_ARBITRUM: ${{ secrets.RPC_ARBITRUM }}
+  RPC_OPTIMISM: ${{ secrets.RPC_OPTIMISM }}
+  RPC_SONIC: ${{ secrets.RPC_SONIC }}
+  RPC_ANVIL: ${{ secrets.RPC_ANVIL }}
+  PRIVATE_KEY: ${{ secrets.ANVIL_PRIVATE_KEY }}
```

I don't see that we use anything other than `RPC_MAINNET` in tests. I think we can remove the others.
```suggestion
  RPC_MAINNET: ${{ secrets.RPC_MAINNET }}
  PRIVATE_KEY: ${{ secrets.ANVIL_PRIVATE_KEY }}
```